bears/typescript/TSLintBear.py

```diff
@@ -4,6 +4,31 @@
 from coalib.bears.requirements.NpmRequirement import NpmRequirement
 from coalib.results.Result import Result
 from coalib.settings.Setting import path
+from coalib.misc.ContextManagers import prepare_file
+
+
+def map_naming_convention_to_tslint(setting):
+    '''
+    This function converts the string ``setting`` (that refers to a naming
+    convention) into a ``tslint`` specific naming convention.
+
+     >>> map_naming_convention_to_tslint('snake')
+     'snake_case'
+     >>> map_naming_convention_to_tslint('camel')
+     'camelCase'
+     >>> map_naming_convention_to_tslint('hyphen')
+     'kebab-case'
+     >>> map_naming_convention_to_tslint(' ')
+     ''
+     '''
+    if setting == 'camel':
+        return 'camelCase'
+    elif setting == 'snake':
+        return 'snake_case'
+    elif setting == 'hyphen':
+        return 'kebab-case'
+    else:
+        return ''
```

Comment:
hey, let's use a dict here to get rid of all the ifs, we still need the function or at least a lambda for the conversion